### PR TITLE
Setup transifex for 7.0

### PR DESCRIPTION
**Pull request by guewen**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,30 +9,35 @@ python:
...
+  - git clone https://github.com/OCA/server-tools ${HOME}/server-tools -b$ {VERSION}
   - printf '[options]\n\nrunning_env = dev' > ${HOME}/.openerp_serverrc
 
 script:
-  - travis_run_flake8
-  - travis_run_tests ${VERSION}
+  - travis_run_tests
 
 after_success:
   coveralls
```
Comment (pedrobaeza) [inline .travis.yml]: The same for this one

File: .travis.yml [hunk 2]
```diff
@@ -9,30 +9,35 @@ python:
...
+  - travis_install_nightly
   - pip install magento
   - pip install beautifulsoup4
-  - git clone https://github.com/OCA/connector ${HOME}/connector -b 7.0
-  - git clone https://github.com/OCA/connector-ecommerce ${HOME}/connector-ecommerce -b 7.0
-  - git clone https://github.com/OCA/sale-workflow ${HOME}/sale-workflow -b 7.0
-  - git clone https://github.com/OCA/e-commerce ${HOME}/e-commerce -b 7.0
-  - git clone https://github.com/OCA/product-attribute ${HOME}/product-attribute -b 7.0
-  - git clone https://github.com/OCA/server-tools ${HOME}/server-tools -b 7.0
+  - git clone https://github.com/OCA/connector ${HOME}/connector -b ${VERSION}
```
Comment (pedrobaeza) [inline .travis.yml]: You can put this in oca_dependencies.txt

**Comment by guewen** [comment]
@pedrobaeza done

**Comment by pedrobaeza** [comment]
Thanks :+1:

**Pull request merged**